docs/wiki/deployment/aws-logging.md

```diff
@@ -25,6 +25,12 @@ When working with AWS, osquery will look for credentials and region configuratio
 
 When logging to Kinesis Streams, the stream name must be specified with `aws_kinesis_stream`, and the log flushing period can be configured with `aws_kinesis_period`.
 
+Configuration flags for kinesis logger:
+
+````
+--aws_kinesis_random_partition_key VALUE         Use random partition keys when sending data to kinesis. Using random values will load balance over stream shards if you are using multiple shards in a stream.  Default is "false".
```

Comment:
I just added a snipit about it in the kinesis description.  I didnt put a reference about it with the other flags since they are all shared flags which this one is not.